services/user.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None
+                ) -> User:
+    user = User(username=username)
```

Comment:
When creating a user, ensure that the username is unique to prevent integrity errors. Consider adding a check or handling exceptions that may arise if a duplicate username is provided.